README.md

```diff
@@ -13,6 +13,35 @@ If you have questions, visit us on irc.freenode.net in #opensuse-factory or
 ask on our mailing list opensuse-factory@opensuse.org
 
 
+### Coding style
+
+The project follows the rules of the parent project
+[os-autoinst](https://github.com/os-autoinst/os-autoinst#how-to-contribute).
+and additionally the following rules:
+
+* Take
+  [example boot.pm](https://github.com/os-autoinst/os-autoinst-distri-example/blob/master/tests/boot.pm)
```

Comment:
linking against master's version LGTM, as that will usually guarantee we have the latest code as reference. Linking to a specific version is not needed at this point in time.